src/mesh.c

```
@@ -389,583 +323,202 @@ int mesh_update_routing(uint16_t source, uint16_t from_node, int16_t rssi, uint8
-    int route_idx = mesh_find_route_index(dest_id);
-    if (route_idx >= 0)
-    {
-        return mesh_state.routes[route_idx].band;
-    }
+/* ============================================================================
+ * Radio callback glue
+ * ============================================================================ */
 
-    /* No specific information - use band with best general performance */
-    uint16_t scores[3] = {0};
+static void mesh_rx_handler(uint8_t* data, size_t len, int16_t rssi, int8_t snr) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'mesh_rx_handler' of convertible types are easily swapped by mistake

(bugprone-easily-swappable-parameters)